WordPress/gutenberg
# Create Block: Update templates to use APIs introduced in WP 6.1

## What?


Now that we have support for `render` in `block.json` coming to WordPress 6.1, we can integrate that into `@wordpress/create-block` to improve the developer experience.

**This PR will have to wait to land until we get closer to the WordPress 6.1 release date (November 1st).**

## How?


Add a new `render` field to `defaultValues` in external templates to let folks define the path to the PHP file that's going to be used with dynamic blocks. I also updated all dynamic variants to use his newly introduced config option.

## Testing Instructions





The following should scaffold a dynamic block that works without the build step.

```bash
npx wp-create-block --variant dynamic --template es5 dynamic-example
```

This is how you can test the version with the build step for the tutorial in Block Editor Handbook:

```bash
npx wp-create-block --no-wp-scripts --variant dynamic --template @wordpress/create-block-tutorial-template dynamic-example-tutorial
cd dynamic-example-tutorial
../node_modules/.bin/wp-scripts build
```
